packages/backend/src/graphql/resolvers/liquidity.ts

```diff
@@ -455,23 +467,31 @@ export const depositOutgoingPaymentLiquidity: MutationResolvers<ApolloContext>['
     }
 
     if (!event.data.debitAmount) {
+      stopTimer()
       throw new Error('No debit amount')
     }
     const outgoingPaymentService = await ctx.container.use(
       'outgoingPaymentService'
     )
+    const stopTimerFund = telemetry.startTimer('fund_payment_ms', {
+      callName: 'outgoingPaymentService.fund'
+    })
     const paymentOrErr = await outgoingPaymentService.fund({
       id: outgoingPaymentId,
       amount: BigInt(event.data.debitAmount.value),
       transferId: event.id
     })
+    stopTimerFund()
+
     if (isFundingError(paymentOrErr)) {
+      stopTimer()
```

Comment:
maybe a finally would be nice here to avoid calling `stopTimer()` multiple times

Reply:
Not used currently, but it should be possible now. I updated the closure returned from startTimer to accept additionalAttributes which will be merged on the ones initially provided to `startTimer`. Usage example added to tests (and tests added for `startTimer` in general - noticed we didnt have any).